selfdrive/car/subaru/carstate.py

```diff
@@ -77,6 +77,8 @@ def update(self, cp, cp_cam, cp_body):
       ret.cruiseState.nonAdaptive = cp_cam.vl["ES_DashStatus"]["Conventional_Cruise"] == 1
       ret.cruiseState.standstill = cp_cam.vl["ES_DashStatus"]["Cruise_State"] == 3
       ret.stockFcw = cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2
```

Comment:
FCW_Cont_Beep is 1, should we add that too?

Reply:
yes, I added FCW_Cont_Beep to stockFcw